internal/store/postgres/migrations/000023_add_audit_log_id_column.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+
+ALTER TABLE "audit_logs"
+ADD COLUMN IF NOT EXISTS "id" UUID DEFAULT uuid_generate_v4();
+
+UPDATE "audit_logs"
+SET "id" = uuid_generate_v4()
+WHERE "id" IS NULL;
```

Comment:
this update command can be skipped as creating the column with default value will automatically populate the uuid